documentation/developer/language/04_arrays_and_tuples.md

```diff
@@ -72,6 +72,30 @@ Leo supports static arrays with fixed size.
 **Dynamic arrays do not exist in Leo**. 
 You cannot change the length of an array once it is initialized.
 
+### Arrays without size 
+Leo allows developers to define arrays, sizes of which will be defined at compile time. Arrays with unknown
```

Comment:
define arrays, sizes of which will be defined at compile time
=>
define arrays without specifying the sizes, which are determined by the compiler